app/models/result.rb

```diff
@@ -0,0 +1,5 @@
+class Result < ApplicationRecord
+    attr_accessor @user_id
```

Comment:
I don't think you need these, they should automatically exist for ApplicationRecord